### PR TITLE
Increases the test coverage analysis to 100% by ensuring that empty classes are ignored, and by extending the test suite for VersionFooter

### LINKED ISSUE #39: Increase the test code coverage to 100%
Currently, the code coverage lies at 64%:

image

- [ ] Code coverage reports should accurately indicate that 100% of the source code base is tested

### PR DESCRIPTION
**Pull request by jrgriffiniii**
Resolves #39

### CODE REVIEW

File: app/models/version_footer.rb [hunk 1]
```diff
@@ -40,7 +40,6 @@ def self.stale?
     else
       @@stale = true
     end
-    @@stale
```
Comment (jrgriffiniii) [inline app/models/version_footer.rb:43]: I believe that this is redundant with the above `if/else` block

File: app/models/version_footer.rb [hunk 2]
```diff
@@ -97,10 +96,12 @@ def self.revisions_logfile
 
   # These assignment methods are needed to facilitate testing
   def self.revision_file=(x)
+    @@stale = true
     @@revision_file = x
   end
 
   def self.revisions_logfile=(x)
+    @@stale = true
```
Comment (jrgriffiniii) [inline app/models/version_footer.rb:104]: Please see https://github.com/pulibrary/pdc_describe/pull/42/files#r838679940

File: app/models/version_footer.rb (hunk 1 above)
Comment (bess) [reply app/models/version_footer.rb:43]: 👍🏻

**Review by bess — APPROVED** [review]
🚀

**Pull request merged**